### ISSUE TITLE
Dockerfile installs amd64-only version of `tini`, not portable to `arm64`

### ISSUE THREAD

**Issue by vielmetti** [ansible/awx#2502]
##### ISSUE TYPE
 - Bug Report

##### COMPONENT NAME

 - Installer

##### SUMMARY

When building awx for `arm64`, it is noted that the Dockerfile has a built-in 
dependency on the `amd64` version of `tini`.

https://github.com/ansible/awx/blob/devel/tools/docker-compose/Dockerfile#L55

```
ADD https://github.com/krallin/tini/releases/download/v0.14.0/tini /tini
```

##### ENVIRONMENT
* AWX version: devel
* Ansible version:  (any)
* Operating System: any on arm64

##### STEPS TO REPRODUCE

On an arm64 system, try to build a working awx.

##### EXPECTED RESULTS

Provided Dockerfile is portable and builds on any architecture.

##### ACTUAL RESULTS

Dockerfile downloads an amd64 version of `tini`, which fails to run on an `arm64` system, even though an arm64 version is available from the download source.

##### ADDITIONAL INFORMATION

This is one of several issues blocking `awx` on `arm64`,  but since it's a tiny one it looks like a good place to start.

**Comment by vielmetti** [comment]
attn @chris-short

**Comment by vielmetti** [comment]
This is one of the several tasks being addressed in #2501

**Comment by james-crowley** [comment]
@ryanpetrello I think this can be closed due to #7607
